docs(hub) update prometheus docs to match version 0.9.0 (#2159)
Kong/docs.konghq.com: app/_hub/kong-inc/prometheus/0.8.x.md

---
name: Prometheus
publisher: Kong Inc.
version: 1.0.0

desc: Expose metrics related to Kong and proxied upstream services in Prometheus exposition format
description: |
    Expose metrics related to Kong and proxied upstream services in [Prometheus](https://prometheus.io/docs/introduction/overview/) exposition format, which can be scraped by a Prometheus Server.

type: plugin
categories:
  - analytics-monitoring

kong_version_compatibility:
    community_edition:
      compatible:
        - 2.0.x
        - 1.5.x      
        - 1.4.x
        - 1.3.x
        - 1.2.x
        - 1.1.x
        - 1.0.x
        - 0.14.x
    enterprise_edition:
      compatible:
        - 1.5.x
        - 1.3-x
        - 0.36-x

params:
  name: prometheus
  service_id: true
  route_id: false
  protocols: ["http", "https", "tcp", "tls", "grpc", "grpcs"]
  dbless_compatible: yes
  dbless_explanation: |
    The database will always be reported as “reachable” in prometheus with DB-less.

---

Metrics are available on the Admin API at the `http://localhost:8001/metrics`
endpoint. Note that the URL to the Admin API will be specific to your

> installation; see _Accessing the metrics_ below.

Reviewer: @gszr Missed this one the first time as well. This should be a link:

```suggestion
installation. See  [Accessing the metrics](#accessing-the-metrics) .
```

kong_nginx_http_current_connections{state="reading"} 0
kong_nginx_http_current_connections{state="total"} 8
kong_nginx_http_current_connections{state="waiting"} 0
kong_nginx_http_current_connections{state="writing"} 1
# HELP kong_nginx_metric_errors_total Number of nginx-lua-prometheus errors
# TYPE kong_nginx_metric_errors_total counter
kong_nginx_metric_errors_total 0
```

### Accessing the metrics

In most configurations, the Kong Admin API will be behind a firewall or would
need to be set up to require authentication, here are a couple of options to
allow access to the `/metrics` endpoint to Prometheus.

1. Kong Enterprise users can protect the admin `/metrics` endpoint with an
   [RBAC user](/enterprise/latest/setting-up-admin-api-rbac) that the
   Prometheus servers use to access the metric data. Access through any
   firewalls would also need to be configured.

2. You can proxy the Admin API through Kong itself then use plugins to limit
   access. For example, you can create a route `/metrics` endpoint and have
   Prometheus access this endpoint to slurp in the metrics while preventing
   others from access it. The specifics of how this is configured will depend

>    on your specific setup. Read the docs [Securing the Admin

Reviewer: ```suggestion
   on your specific setup. For details, see [Securing the Admin
```

>    API](https://docs.konghq.com/latest/secure-admin-api/#kong-api-loopback) for

Reviewer: ```suggestion
   API](https://docs.konghq.com/latest/secure-admin-api/#kong-api-loopback).
```

>    details.

Reviewer: ```suggestion
 
```

3. Finally, you could serve the content on a different port with a custom server
   block using a [custom Nginx
   template](/latest/configuration/#custom-nginx-configuration) with Kong.

>     The following block is an example custom nginx template which can be used

Reviewer: ```suggestion
    The following block is an example custom nginx template that can be used
```

https://www.grammarly.com/blog/which-vs-that/
